mkosi/machine.py

```diff
@@ -171,7 +171,10 @@ def setUpClass(cls) -> None:
         if os.getuid() != 0:
             raise unittest.SkipTest("Must be invoked as root.")
 
-        cls.machine = Machine(cls.args)
+        try:
+            cls.machine = Machine(cls.args)
```

Comment:
So this approach means we won't build any of the images that aren't bootable anymore. In the CI we, first used the "build" verb and then used the "boot" or "qemu" verbs to boot the image in a second step.

What we can do to fix this is to move all the "not supported" checks involving verbs out of load_args() and into their respective run_xxx() functions. This means we'll still build the image but will raise MkosiNotSupported only when trying to boot the image.